### PR TITLE
add rational activation function

### PR DESCRIPTION
**Pull request by seviyorm**
I made a new activation function named "rational" into keras.

This activation function looks similar to sigmoid. (see https://c.mql5.com/18/20/af__1.gif)
However this doesn't include exponential calculation which has expensive complexity.

When the absolute value of input is bigger than about 3~4, the gradient of this activation function is higher than sigmoid or tanh function.
Therefore I think it will prevent gradient vanishing problem.

### CODE REVIEW

**Comment by fchollet** [comment]
Do have a conference paper or journal paper exploring the benefits / tradeoffs of this activation in different types of nets?

**Comment by the-moliver** [comment]
That's just the softsign activation which is already included.

**Comment by fchollet** [comment]
Thanks for pointing this out.
